src/add-bibtex-reference.command.ts

```diff
@@ -55,7 +55,9 @@ export async function registerAddBibTexReferenceCommand () {
 
             } catch (e) {
                 console.log(e.message);
-                await showMessageBox(ERROR_PARSING_FAILED);
+                await joplin.views.dialogs.showMessageBox(
+                    ERROR_PARSING_FAILED + e.message
```

Comment:
So that the message box content is a bit more readable, could you add two new lines between ERROR_PARSING_FAILED and e.message?